java/openjdk/Portfile

```diff
@@ -595,6 +616,30 @@ subport openjdk17-oracle {
     worksrcdir   jdk-${version}.jdk
 }
 
+subport openjdk17-sap {
+    # https://github.com/SAP/SapMachine/releases/tag/
```

Comment:
There is no page at this URL. I suggest using https://github.com/SAP/SapMachine/releases/ instead.